src/components/pages/home/calltoaction.tsx

```diff
@@ -6,7 +6,7 @@ import { useRouter } from 'next/navigation';
 const CallToAction = () => {
   const router = useRouter();
   return (
-    <div className='m-[12px] px-6 py-12 md:px-16'>
+    <div className='px m-[2px] py-12 md:px-16'>
```

Comment:
Looks good @R-pradeep2005 I really love how it aligned it very well. 
<img width="387" height="883" alt="Screenshot 2025-08-30 at 6 43 40 AM" src="https://github.com/user-attachments/assets/9ce2f930-ebb0-421c-8604-e93c44639ca7" />

One thing I noticed about this is that in small screen sizes, it has a drastic change in the cta section with no spacing on the sides. 

Can you please look into this and fix this aswell, would be more than happy to merge this once it has the updated changes. Thanks a ton

Cheers mate :)